gcs-action/action.yml

```diff
@@ -0,0 +1,26 @@
+name: 'GCS Action'
+description: 'Enable easy upload of files and directories to GCS'
+author: 'GoogleCloudPlatform'
+inputs:
+  service_account_key:
+    description: 'GCP service account key'
+    required: true
+  folder-to-upload:
+    description: 'Folder to be uploaded'
+    required: false
+  file-to-upload:
+    description: 'File to be uploaded'
+    required: false
+  bucket-name:
+    description: 'GCS Bucket Name'
```

Comment:
In what format? `gs://`?

Reply:
I think `destination` might be a better thing here, in which the user specifies the bucket and (optionally) folder like

```yaml
destination: my-bucket/subfolder
```

WDYT?

Reply:
I think this makes sense and is cleaner. Any thoughts @craigdbarber

Reply:
Using the destination of form my-bucket/prefix